File: api/src/test/java/net/blueshell/api/config/ValidatorConfig.java
package net.blueshell.api.config;

import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;
import org.springframework.validation.beanvalidation.LocalValidatorFactoryBean;

/**
 * Validator bean enabling Jakarta Bean Validation in tests.
 */
@Configuration
public class ValidatorConfig {
    @Bean
    public LocalValidatorFactoryBean validator() {
        return new LocalValidatorFactoryBean();
    }
}

File: api/src/test/kotlin/net/blueshell/api/config/ValidatorConfig.kt
package net.blueshell.api.platform.config

import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration
import org.springframework.validation.beanvalidation.LocalValidatorFactoryBean

/**
 * Validator bean enabling Jakarta Bean Validation in tests.
 */
@Configuration
class ValidatorConfig {
    @Bean
    fun validator(): LocalValidatorFactoryBean = LocalValidatorFactoryBean()
}
